### PR TITLE
Test server fix

### PR DESCRIPTION
**Pull request by Crash1212**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "lint:fix": "eslint . --fix",
     "test-server": "cross-env NODE_ENV=test nodemon server.js",
     "test": "jest --coverage --detectOpenHandles ",
-    "generate": "graphql-codegen"
+    "generate": "graphql-codegen",
+    "ci": "start-server-and-test test-server http://localhost:6000 test"
```
Comment (Alan18081) [inline package.json:19]: Please, rename the script to the `test-full` and take port number from the environment variable

**Review by bandvov — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Crash1212) [reply package.json]: fixed

File: package-lock.json [hunk 2]
```diff
@@ -2646,9 +2646,9 @@
       }
```
Comment (Alan18081) [inline package-lock.json:2646]: Please, remove from commit changes to `package-lock.json`, it can create unnecessary issues

File: package.json [hunk 3]
```diff
@@ -1,87 +0,0 @@
-{
```
Comment (Alan18081) [inline package.json:1]: Please, revert deleting the `package.json`

File: package-lock.json (hunk 2 above)
Comment (Crash1212) [reply package-lock.json]: fixed

File: package.json (hunk 3 above)
Comment (Crash1212) [reply package.json]: fixed

**Pull request merged**